06-building-a-program-start-to-finish/solving-the-problem/output/InvertedIndexUtility.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-unsed-vars */
+/* eslint-disable no-undef */
+
+/**
+ * Utility class for InvertedIndex class
+ * @class InvertedIndexUtility
+ */
+class InvertedIndexUtility {
+/**
+ * Checks input to see if it conforms to specific standards.
+ * @param {Array} file - Book file to be validated
+ * @returns {boolean} true/false - returns validation status.
+ * @memberOf InvertedIndex
+ */
+  static validateInput(file) {
+    if (Array.isArray(file) && file.length) {
+      const books = Object.keys(file);
+      for (let i = 0; i < books.length; i += 1) {
+        if (file[i].text && file[i].title) {
```

Comment:
I'm not sure you have this loop written right.

You're ultimately stopping the execution after the first element in the array.